### ISSUE TITLE
error in test_Sony.py

### ISSUE THREAD

**Issue by PeterStone007** [cchen156/Learning-to-See-in-the-Dark#34]
after download and extract raw Sony data in dir dataset/Sony.
run test_Sony.py
got error at line 151:
 im = raw.postprocess(use_camera_wb=True, half_size=False, no_auto_bright=True, output_bps=16)

rawpy._rawpy.LibRawFatalError: b'Unsufficient memory'
could you have met this? 
Appreciate for answer.

**Comment by cchen156** [comment]
Never met this error before. How large is your RAM?

**Comment by PeterStone007** [comment]
32Gb. I also run the code with another pc with 64Gb, got the error again.. it so confuse me.

**Comment by PeterStone007** [comment]
@cchen156

**Comment by cchen156** [comment]
I am not sure what the problem is because I am not able to reproduce the error. Can you try output_bps=8?

**Comment by danielberbece** [comment]
I received the same error. I changed the 'half_size' parameter to True and it worked. I don't understand why, because I have enough RAM on my computer.

**Comment by ninesun127** [comment]
My GPU memory is 12GB,the same error as you, i follow the danielberbece‘s advice,but the result is absolutely wrong

**Comment by cchen156** [comment]
Maybe there is a bug in the new version of Rawpy. Which version did you use? @PeterStone007 @danielberbece @ninesun127

**Comment by cchen156** [comment]
I am using 0.10.1 and I cannot reproduce the error.

**Comment by ninesun127** [comment]
0.11.0
it is a really terrible thing~

**Comment by ninesun127** [comment]
@cchen156  yeah,when i download the rawpy  0.10.1, there is no error when running。genuinely thanks for your patient reply.

as you said,there is a bug in the new version of Rawpy.

**Comment by danielberbece** [comment]
yes, downgrading from 0.11.0 to 0.10.0 solved the issue for me too
